cmd/ci-op-configs-mirror/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/interrupts"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+	"github.com/openshift/ci-tools/pkg/promotion"
+)
+
+const (
+	privatePromotionIS = "4.4-private"
```

Comment:
Huh, I know this is just a temporary measure, but either 1) name this something that clearly communicates it's not a real destination for promotion (it should definitely not contain any version substring `4.4`) or 2) dynamically create the name by appending the `-private` suffix to whatever name is in the original. This way it's confusing.